src/FffGcodeWriter.cpp

```diff
@@ -1785,7 +1789,7 @@ bool FffGcodeWriter::partitionInfillBySkinAbove(Polygons& infill_below_skin, Pol
     const coord_t infill_skin_overlap = mesh.settings.get<coord_t>((part.insets.size() > 1) ? "wall_line_width_x" : "wall_line_width_0") / 2;
     const Polygons infill_below_skin_overlap = infill_below_skin.offset(-(infill_skin_overlap + tiny_infill_offset));
 
-    return ! infill_below_skin_overlap.empty() && ! infill_not_below_skin.empty();
+    return !infill_below_skin_overlap.empty() && !infill_not_below_skin.empty();
```

Comment:
Ah, I've been thinking that `!variable` was the most common way of writing this. Again our code style rules are silent about it so there is nothing really enforced (to prevent endless discussions on code style as well). I think it's fine for details like this to not enforce it. But as you can see I also have a natural tendency to make those things orderly... :stuck_out_tongue: 

The software wide meeting "Code & Cookies" in which these code style rules were originally created is no longer. It was split up into meetings with the individual teams (hence "Cura & Cookies") and semi-replaced by an architecture guild which should discuss these things. Code style is company-wide so it's a bit hard to make these refinement sometimes :(

Reply:
I find that the extra space increases readability, 'not' being an important distinction you don't want to read-over, but I don't care _that_ much.

Reply:
Leave it as is